proxy_wasm_api.h

```diff
@@ -30,6 +30,10 @@
 #include <utility>
 #include <vector>
 
+#include "proxy_wasm_common.h"
+#include "proxy_wasm_enums.h"
+#include "proxy_wasm_externs.h"
```

Comment:
This breaks plugins (see: https://github.com/envoyproxy/envoy/pull/16897) due to this file being included from `null_plugin` namespace... I'm looking into it.

Reply:
I spent way too much time trying to figure this out, so I reverted those changes.